packages/alfa-rules/src/experimental.ts

```diff
@@ -1,3 +1,4 @@
 export { default as ER62 } from "./sia-er62/rule";
 export { default as ER87 } from "./sia-er87/rule";
+export { default as R82 } from "./sia-r82/rule";
```

Comment:
Is this a breaking change or has `R82` not been used yet?

Reply:
R82 is not used yet (because it asks questions about every form field, which is too much…)
(plus it already uses questions' context which Dory/Nemo/DB can't handle yet)